framework/requirements.txt

```diff
@@ -82,3 +82,4 @@ websocket-client==0.57.0
 Werkzeug==2.0.2
 xmltodict==0.12.0
 yarl==1.6.3
+python-json-logger==2.0.2
```

Comment:
I recommend that you use the QA dependencies test in order to see possible vulnerabilities. Post the result in a PR update. https://github.com/wazuh/wazuh-qa/tree/master/tests/scans/dependencies